windows/jitsi-meet/index.html

```diff
@@ -0,0 +1,15 @@
+ <!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="../../css/welcome.css">
+        <script src="https://meet.jit.si/external_api.js"></script>
```

Comment:
this needs to be based on the domain. Instead of loading it here, lazy load it with https://github.com/feross/load-script2